<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quickaccessPreviewConfig</class>
 <widget class="QWidget" name="quickaccessPreviewConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="previewBox">
     <property name="text">
      <string>Show previews</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="previewLabel1">
     <property name="text">
      <string>Preview plugins:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pluginLayout">
     <item>
      <widget class="QLabel" name="previewLabel2">
       <property name="text">
        <string>Here you can enable/disable the plugins.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
